lib/index.ts

```diff
@@ -98,10 +100,15 @@ export async function inspect(
     let callGraph: CallGraph | undefined;
     if (options.reachableVulns) {
       debug(`getting call graph from path ${targetPath}`);
-      callGraph = await javaCallGraphBuilder.getCallGraphMvn(
-        path.dirname(targetPath),
-      );
-      debug('got call graph successfully');
+      try {
+        callGraph = await javaCallGraphBuilder.getCallGraphMvn(
+          path.dirname(targetPath),
+        );
+        debug('got call graph successfully');
+      } catch (err) {
+        debug('call graph error: ', err);
+        throw new CallGraphError(err.message, err);
```

Comment:
`throw new CallGraphError(err);`
and inside CallGraphError distruct message from err

Reply:
This is an implementation detail that I don't want to enforce currently in this class.